src/Database/PostgreSQL/Simple/Migration.hs

```diff
@@ -101,11 +121,32 @@ runMigrations
     -> [MigrationCommand]
        -- ^ The commands to run
     -> IO (MigrationResult String)
-runMigrations verbose con commands =
-    sequenceMigrations [runMigration (MigrationContext c verbose con) | c <- commands]
+runMigrations = runMigrationsA defaultLogWrite
+
+-- | A version of 'runMigrations' which gives you control of where the log
+-- messages are sent to.
+runMigrationsA
```

Comment:
I know I said `runMigrationsA` in the previous review, but this could probably have a more descriptive name? I'm terrible at naming things.

Reply:
@MasseR Ha, I have the same problem. I couldn’t come up with something good. I could name it as `runMigrationsWithCustomLogWriteFn` but I’m hesitating to add this.